src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+            type="text"
+            name="Name"
+            autocomplete="off"
+            required
+          >
+        </label>
+        <label class="label">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+        <label class="label">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+          >
+        </label>
+        <label class="label">
+          I accept the term of the agreement:
+          <input
+            type="checkbox"
+            name="terms"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="label">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          >
+        </label>
+        <label class="label">
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="6"
+          maxlength="32"
+          >
+        </label>
```

Comment:
[checklist p.2](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

I have already submitted this comment but it seems you did not fix it in all the places. Please if I leave some code-style comments - make sure to double-check and fix them in all the usage places.

Reply:
Could you elaborate on the issue with the placeholder, please?